Make js driver to use external python script to control neo4j server

zhenlineo opened the pull request.

pontusmelke commented on gulpfile.js:
```diff
@@ -247,28 +195,42 @@ gulp.task('set', function() {
+gulp.task('start-neo4j', function() {
+    var neorunArgs;
+    //var neorunArgs = minimist(process.argv.slice(2), { string: 'opts' }).opts;
+    if (neorunArgs == null)
```
How can `neoRunArgs` not be `null` here? Typically the more ideomatic way is doing

```
neoRunArgs = neoRunArgs || '-v 3.0.1 -p neo4j'
```

pontusmelke commented on neokit:
```diff
@@ -0,0 +1 @@
+Subproject commit d2059f1a4bfc8775697937eea5285b69864f2b7f
```
You should probably not commit this?

pontusmelke commented on test/v1/tck/steps/util.js:
```diff
@@ -354,48 +354,38 @@ function changeCertificates(keyFile, certFile) {
 
-function restart(callback) {
+function restart() {
+  // how to sequentially execute these two methods????
```
I think the easiest way is to use timeouts as we did before.

Commit:
Change the java driver to use neokit to manage server start

Commit:
Default to use 3.0-nightly for testing.

Commit:
Make runTests script to accept args for neorun
To run tests against 3.0 nightly: `runTests.sh`
To run tests against 3.1 nightly: `runTests.sh '-n 3.1 -p neo4j'`

The first arg after runTests is the arg that will be passed to neorun.
JS driver runs tests with auth enabled, so we need to provide `-p neo4j` for changing the default password.

Commit:
Update neokit to both support python v2 and v3

Commit:
Made the powershell script to accept args
Made the powershell script to stop server even after ctrl+C
